coch.py

```diff
@@ -0,0 +1,53 @@
+import os
+import sys
+import argparse
+
+def main(mode='rand_sound'):
+    mode = mode.lower()
+
+    from pycochleagram import cochleagram as cgram
+    from pycochleagram import erbfilter as erb
+    from pycochleagram import utils
+    from random import choice
+    from pycochleagram.demo import demo_human_cochleagram_helper
+
+    from os.path import dirname, join, realpath
+    from pydub import AudioSegment
+    # from argparse import argumentparser
+  #AUDIO PROCESSING SECTION: MANIPULATES MP3 FILES AND CONVERTS TO WAV, SLICES $
+    DEMO_PATH = join(dirname(realpath(__file__)) #'/media/sf_acm_mirum_tempo' THE DATASET LINK MUST BE PUT HERE
+    if mode == 'rand_sound':
+        rfn=[]
+        soundname=[]
+        file_extension=[]
+    #mpg123 -w train.append(f.wav) f 
+        path, dirs, files = next(os.walk(DEMO_PATH))
+        file_count = len(files)
+        for i in range(0,file_count):
+            rfn.append(choice([os.path.join(DEMO_PATH,f)for f in os.listdir(DEMO_PATH) if f.endswith('.mp3')]))
+            soundname.append(AudioSegment.from_mp3(rfn[i]))
+            clips = soundname[i][:10000]
```

Comment:
Why should it be 10000 here?

Reply:
this is the time in milliseconds that I define to clip an audio segment for cochleagram dimensions. I need 256 time bins of 0.5 ms in length approximately. I can use 10 secs of audio and I have a zero-padding function to shape my input.